packages/zosfiles/src/methods/download/Download.ts

```diff
@@ -360,7 +360,7 @@ export class Download {
                 }
 
                 if (options.directory == null) {
-                    if (dataSetObj.dsorg === "PO" || dataSetObj.dsorg === "PO-E") {
+                    if (dataSetObj.dsorg && dataSetObj.dsorg.startsWith("PO")) {
```

Comment:
you could combine this using the `?.` operator 😋 
```suggestion
                    if (dataSetObj.dsorg?.startsWith("PO")) {
```